docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -24,45 +24,48 @@ import Link from '@docusaurus/Link';
 - [UID2Manager API](#uid2manager-api)
   -  [Functions](#functions)
   -  [Variables](#variables)
-- [Android Initialization](#android-initialization)
 - [Code Samples](#code-samples) -->
 
-You can use the UID2 SDK for iOS to facilitate the process of establishing client identity using UID2 and retrieving advertising tokens on iOS devices.
+You can use the UID2 SDK for iOS to facilitate the process of performing the following activities:
+
+- Generating or establishing client identity using UID2.
+- Retrieving advertising tokens on iOS devices for bid stream use.
+- Automatically refreshes UID2 tokens.
```

Comment:
fixed

Reply:
I think we should say "refreshing" instead of "refreshes".

> You can use the UID2 SDK for iOS for the following: Automatically refreshes UID2 tokens.

Doesn't sound right. Same issue is in the Android docs.

Reply:
fixed